### PR TITLE
Updated dockerfile, docker-compose

### PR DESCRIPTION
**Pull request by wniedzwiedz**
Using COPY --from:builder gives this:

Step 5/6 : COPY --from=builder ./target/spring-boot-sample-data-jdbc-2.1.13.BUILD-SNAPSHOT.jar app.jar
COPY failed: stat /var/lib/docker/overlay2/f36418eb7a7053035de42cf1f0f47459b0395783ffbd3d3f1c739fd3a8d7ced3/merged/target/spring-boot-sample-data-jdbc-2.1.13.BUILD-SNAPSHOT.jar: no such file or directory

### CODE REVIEW

File: spring-boot-samples/spring-boot-sample-data-jdbc/Dockerfile [hunk 1]
```diff
@@ -1,13 +1,8 @@
...
-ARG USER
+FROM openjdk:13-jdk-alpine AS builder
 
-FROM openjdk:13-jdk-alpine AS build
-
-CMD ["clean", "-f", "pom.xml"]
-CMD ["install", "-f", "pom.xml"]
-RUN echo "User: $USER"
-RUN echo "Pass: $PASS"
+CMD ["mvn", "clean", "install"]
```
[the remark this answers is not in the document]
Comment (rkhozinov) [reply spring-boot-samples/spring-boot-sample-data-jdbc/Dockerfile]: btw, try this tool to check container layers optimisation 

https://github.com/wagoodman/dive

File: spring-boot-samples/spring-boot-sample-data-jdbc/docker-compose.yml [hunk 2]
```diff
@@ -5,8 +5,7 @@ services:
...
-    build:
-      context: .
-      args: 
-        - user:qwerty
-        - pass:123
\ No newline at end of file
+    image: postgres
+    restart: always
+    environment:
+      env_file: ./user.env
```
Comment (rkhozinov) [inline spring-boot-samples/spring-boot-sample-data-jdbc/docker-compose.yml]: 👍

File: spring-boot-samples/spring-boot-sample-data-jdbc/Dockerfile [hunk 3]
```diff
@@ -1,13 +1,8 @@
...
+COPY ./target/spring-boot-sample-data-jdbc-2.1.13.BUILD-SNAPSHOT.jar .
 
 FROM openjdk:13-jdk-alpine AS run
-COPY ./target/spring-boot-sample-data-jdbc-2.1.13.BUILD-SNAPSHOT.jar ./app.jar
+COPY ./target/spring-boot-sample-data-jdbc-2.1.13.BUILD-SNAPSHOT.jar app.jar
```
[the remark this answers is not in the document]
Comment (wniedzwiedz) [reply spring-boot-samples/spring-boot-sample-data-jdbc/Dockerfile]: Step 5/6 : COPY --from=builder ./target/spring-boot-sample-data-jdbc-2.1.13.BUILD-SNAPSHOT.jar app.jar
COPY failed: stat /var/lib/docker/overlay2/f93d28121eaca3da8a4191db45c634f3423ae98a726287a2550b51f18bb593a8/merged/target/spring-boot-sample-data-jdbc-2.1.13.BUILD-SNAPSHOT.jar: no such file or directory

File: spring-boot-samples/spring-boot-sample-data-jdbc/Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (wniedzwiedz) [reply spring-boot-samples/spring-boot-sample-data-jdbc/Dockerfile]: Step 2/6 : RUN mvn clean install
 ---> Running in 937979afc5ad
/bin/sh: mvn: not found
The command '/bin/sh -c mvn clean install' returned a non-zero code: 127
C6863:spring-boot-sample-data-jdbc wniedzwiedz$

**Pull request closed without merging**